magentoerpconnect/product.py

```diff
@@ -631,6 +631,16 @@ def _must_skip(self):
 
         :returns: None | str | unicode
         """
+        prefix = self.env['ir.config_parameter'].get_param(
+            'outlet_product_prefix')
+        if self.magento_record['sku'] and \
+                self.magento_record['sku'].startswith(prefix + '-'):
+            return 'Outlet products are mapped to real products and will'\
```

Comment:
Nicer within parenthesis instead of line continuation

```
return ('Outlet products are mapped to real products and will'
    ' not be imported in ERP. All products with prefix')